### PR TITLE
Add Sentry

### PR DESCRIPTION
**Pull request by fumimowdan**
-------

## Description
Add sentry to monitor the service

## Trello Card Link
[card](https://trello.com/c/Rm6A8G8L/6-s-collect-errors-according-to-dfe-standards-with-sentry)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -70,3 +70,6 @@ group :development do
   gem "binding_of_caller"
   gem "web-console"
 end
+
+gem "sentry-ruby", "~> 5.10"
```
Comment (raul-gracia) [inline Gemfile:74]: ❓ Do we need `sentry-ruby` if we are already including `sentry-rails` ? since `sentry-rails` already depends on `sentry-ruby? `

**Review by raul-gracia — APPROVED** [review]
✅ LGTM

File: Gemfile (hunk 1 above)
Comment (fumimowdan) [reply Gemfile]: removed explicit sentry-ruby dependency

**Comment by fumimowdan** [comment]
[[S] Collect errors according to Dfe standards with `Sentry`](https://trello.com/c/Rm6A8G8L/6-s-collect-errors-according-to-dfe-standards-with-sentry)

**Pull request merged**